docs/reference/api/generated-api-html/index.html

```diff
@@ -6124,6 +6124,51 @@ <h3 class="field-label">Example data</h3>
       "logLines" : [ "logLines", "logLines" ]
     },
     "succeeded" : true
+  },
+  "catalogDiff" : {
+    "transforms" : [ {
+      "streamDescriptor" : {
+        "name" : "name",
+        "namespace" : "namespace"
+      },
+      "transformType" : "add_stream",
+      "updateStream" : [ {
+        "updateFieldSchema" : { },
+        "fieldName" : [ "fieldName", "fieldName" ],
+        "addField" : { },
+        "transformType" : "add_field",
```

Comment:
Do we have a list of the transforms somewhere and which ones are breaking? I don't remember seeing the details.

Reply:
the details are in the tech design: https://docs.google.com/document/d/1u333RpXQ1yHv3TK_4cjDjvGnaJeHtvkjxrq3oY5IT1E/edit#heading=h.uqbexni35qq1

the catalog diff functionality was pre-existing, this PR just adds them to the openapi.yaml. I'm actually not sure about the difference between openapi.yaml and openapi/config.yaml - this object was already in openapi/config.yaml, so maybe this addition isn't needed